databases/week1/hyf_lesson1_db1hw.sql

```diff
@@ -0,0 +1,88 @@
+-- 1. Find out how many tasks are in the task table
+SELECT
+  COUNT(title)
+from
+  task;
+-- 2.Find out how many tasks in the task table do not have a valid due date
+SELECT
+  COUNT(title)
+from
+  task
+where
+  due_date IS NULL;
+-- 3.Find all the tasks that are marked as done
+SELECT
+  task.title,
+  status.name
+from
```

Comment:
You should be using a JOIN here, if nothing else it's best practice :)

Reply:
ok yes, not it is clear !